cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/configuration/PricingConfig.java

```diff
@@ -0,0 +1,33 @@
+package com.sequenceiq.cloudbreak.cloud.configuration;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+import javax.inject.Inject;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import com.google.common.collect.Maps;
+import com.sequenceiq.cloudbreak.cloud.PricingCache;
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+
+@Configuration
+public class PricingConfig {
+
+    @Inject
+    private Optional<List<PricingCache>> pricingCaches;
+
+    @Bean
+    public Map<CloudPlatform, PricingCache> pricingCacheMap() {
+        if (pricingCaches.isPresent()) {
+            Map<CloudPlatform, PricingCache> pricingCacheMap = Maps.newHashMap();
+            for (PricingCache pricingCache : pricingCaches.get()) {
+                pricingCacheMap.put(pricingCache.getCloudPlatform(), pricingCache);
+            }
+            return pricingCacheMap;
+        }
+        return Map.of();
```

Comment:
You can simplify it 

return pricingCaches.orElse(Collections.emptyList()).stream().collect(Collectors.toMap(pricingCache-> pricingCache.getCloudPlatform(),Function.identity()));